ros_gz_bridge/launch/ros_gz_bridge.launch

```diff
@@ -1,11 +1,13 @@
 <launch>
+  <arg name="name" default="ros_gz_bridge" />
```

Comment:
This should be a required argument, I.e, it should not have a default value. If we do want a default value, we should consider using [`anon`](https://github.com/ros2/launch/blob/aed025e04e0b143362c69bf29a67b2ffff4c59ee/launch/launch/substitutions/anon_name.py#L26)

Reply:
I made it required to avoid user errors. [9dc4f26](https://github.com/gazebosim/ros_gz/pull/556/commits/9dc4f26ef75739ec552c365bffbe00e9a16e8104)